content/doc/upgrade-guide/2.107.adoc

```diff
@@ -60,9 +60,16 @@ It is no longer necessary to provide a CSRF crumb when sending an HTTP request w
 
 This is a list of confirmed regressions introduced in this release.
 
+// TODO: post known regressions for all releases?
+// JENKINS-48770 - introduced in 2.89
 // JENKINS-48821 is not needed here as it was introduced in 2.89.x due to backport
+// JENKINS-49588 - unconfirmed, maybe introduced in 2.107
```

Comment:
@oleg-nenashev Why add commented out text?

Reply:
Knowing which issues were considered but ultimately not listed can guide future updates of this list.